### ISSUE TITLE
Serious issue with Win98 setup

### ISSUE THREAD

**Issue by DSN9990** [joncampbell123/dosbox-x#2684]
I am having a serious issue with my first Win98 setup I ever made on DOSBox-X. The problem is that it crashes with a "Illegal Operation" error when it tries to install the default monitor settings: 

![guest os_002](https://user-images.githubusercontent.com/69558812/126013373-feb1e1df-6214-4a88-9627-5bff1c72155e.png)

![guest os_003](https://user-images.githubusercontent.com/69558812/126013395-11fd9b8d-fbda-4895-a5cd-56937435e2a7.png)

I first used normal core, then dynamic_x86 core. And then I tried disabling APM bios... neither seem to help with this crash. Then, after the error message is shown, I get stuck on a empty blank screen, see below:

![guest os_004](https://user-images.githubusercontent.com/69558812/126013742-949406d2-e98f-4eeb-9d0a-dc17471a362b.png)

Is there any solution to this? Perhaps this is a bug with DOSBox-X?

**Comment by Wengier** [comment]
I have done a fresh installation of Windows 98, using the latest code and the exact same config file as mentioned in the Wiki, and it boots to the desktop just fine:

![image](https://user-images.githubusercontent.com/8216923/126029475-45f346e4-5ef0-4ca6-b84d-b0c2d02fb19b.png)

So I think something is wrong either with the Windows 98 installation steps, or the Windows 98 setup disks.

**Comment by DSN9990** [comment]
@Wengier I also followed the wiki guide, and I am not sure what this error does mean. Also forgot to say, I'm using Windows 98 FE and not SE at this time. What ISO image/Windows 98 version are you using? I used an ISO image from WinWorld, though.

**Comment by rderooy** [comment]
I occasionally try a WIn98 install to check for regressions, but typically I try the 98SE version. Can you check with 98SE? Also, does your config have any other settings, other then the once mentioned on the wiki?

Also, you don't mention which dosbox-x binary your using. SDL1, SDL2, Linux, Windows VS or Windows MinGW, 32bit or 64bit.

**Comment by DSN9990** [comment]
@rderooy I am using MinGW64 SDL2 dosbox-x and the only other setting I have is `apmbios=false` here, to avoid certain crashes in Win98/ME as stated by @joncampbell123 earlier. Indeed, Windows 98 First Edition now works fine here, at least for now, see below:

![guest os_005](https://user-images.githubusercontent.com/69558812/126040906-28201008-c827-4a73-8c05-09837e997121.png)

**Comment by DSN9990** [comment]
Forgot one more thing, I followed method 1 from the wiki install guide, which is the CD-ROM boot method.

**Comment by rderooy** [comment]
@DSN9990 what did you do to make it work? I have been testing with apm support enabled, and have not encountered crashes.

My quick testing actually uses method 2 with a response template, such that I don't need to sit there answering every question.

**Comment by DSN9990** [comment]
@rderooy I'm also having some slowdowns in certain programs on my end. Perhaps that is because of the `use dynamic core with paging on=auto` setting, which will enable itself if a guest system is detected?

**Comment by DSN9990** [comment]
@rderooy I'm also having some slowdowns in certain programs on my end. Perhaps that is due to the `use dynamic core with paging on=auto` setting, which will enable itself if a guest system is detected?

**Comment by Wengier** [comment]
@DSN9990 Yes, for dynamic x86 core, the setting ``use dynamic core with paging on=auto`` will enable itself if a guest system is detected will enable itself if a guest system is detected as you said.

**Comment by DSN9990** [comment]
@Wengier I got a lot of crashes on Win98FE so I will give up for now. Soon I will try to find some time to test Win98SE though...
